source/wp-content/themes/wporg-parent-2021/patterns/two-columns-with-cta.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Title: 2 Columns with CTA
+ * Slug: wporg-parent-2021/two-columns-with-cta
+ * Categories: columns, buttons
+ */
+
+?>
+<!-- wp:columns {"align":"full","style":{"border":{"top":{"color":"var:preset|color|light-grey"}}},"className":"is-style-default"} -->
+<div class="wp-block-columns alignfull is-style-default" style="border-top-color:var(--wp--preset--color--light-grey)"><!-- wp:column {"width":"100%","style":{"spacing":{"padding":{"top":"80px","right":"100px","bottom":"100px","left":"100px"}}}} -->
```

Comment:
This border color doesn't match the design but I've assumed we will adjust this in the theme settings?